test/screenshot/golden.json

```diff
@@ -1077,6 +1077,123 @@
       "desktop_windows_ie@11": "https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/10/23/19_08_56_617/spec/mdc-select/mixins/shape-radius.html.windows_ie_11.png"
     }
   },
+  "spec/mdc-snackbar/classes/baseline-with-action.html": {
```

Comment:
Snackbar is being clipped in iPhone 5 viewport.

Reply:
Wow, good catch! 👍

That's due to the default `min-width` value of `344px` from the spec:

https://material.io/design/components/snackbars.html#spec

![image](https://user-images.githubusercontent.com/409245/48295921-8cbef000-e445-11e8-8bdc-19fdf57e8df6.png)

I'll follow up with Design on this.

(OOC, why did you think to test it on an iPhone 5? Never would have occurred to me 😝)

Reply:
I was testing if snackbar snaps to screen edges for smaller browsers 🙂